students/seyoung/users/views.py

```diff
@@ -44,4 +45,23 @@ def post(self, request):
                 date_of_birth = input_birthdate,
         )
 
-        return JsonResponse({"message": "SUCCESS"}, status=201)
\ No newline at end of file
+        return JsonResponse({"message": "SUCCESS"}, status=201)
+
+class UserSignInView(View):
+    def post(self, request):
+        try:
+            input_data     = json.loads(request.body)
+            input_email    = input_data["email"]
+            input_password = input_data["password"]
+            
+            if not User.objects.filter(email = input_email, \
+                password = input_password).exists():
```

Comment:
굳이 줄바꿈이 필요한 것 같지 않습니다!
```suggestion
            if not User.objects.filter(email = input_email, password = input_password).exists():
```

Reply:
알겠습니다